ECHO-Sunrise.ipynb

```diff
@@ -340,18 +258,26 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "ranked = my_prog_data.groupby([\"Index\", \"FAC_NAME\", \"FAC_LAT\", \"FAC_LONG\"])[[a]].agg(att_column[program.name][\"agg\"])\n",
-    "ranked.reset_index(inplace=True)\n",
-    "ranked = ranked.set_index(\"Index\")\n",
-    "ranked.to_csv(\"facilities_ranked.csv\")\n",
-    "ranked.sort_values(by=a, ascending=False)"
+    "%run utilities.py # Load the right style parameters\n",
+    "plt.figure(figsize=(20,10))\n",
+    "plt.bar(bars.index, bars[program.agg_col])\n",
+    "plt.bar(bars.index, bars[program.agg_col + \" in this District\"])\n",
+    "plt.title(program.name)\n",
+    "plt.xlabel( 'Year' )\n",
+    "plt.ylabel( program.unit )\n",
+    "plt.legend([\"Statewide\", \"In %s\" %(select_region_widget.value)])\n",
+    "plt.show()   \n",
+    "\n",
+    "bars.to_csv(\"trends-\"+program.name+\"-MA-\"+select_region_widget.value+\".csv\")"
```

Comment:
Well shoot.

This is a great observation and I'm very sorry I missed it - the legend should be labeled "Rest of the state" as you say instead of statewide because the CD is a subset of that.

>Or actually, what if there were just bars for the CD and lines showing mean, median, and mode of districts in the state? Feel like that would be a more meaningful comparison "is my district unusual"

Great idea! This may take some time to do...I will try to pull it off.

Reply:
The idea to look at statewide data came from the Sunrise Notebook Context doc https://docs.google.com/document/d/11gMHQXYyyJYE1o9f1PjQmrnaGCYXvBd0nkU4SncmJsY/edit last week and necessitated a major rewrite of the notebook for this important goal. Other folks have mentioned wanting to assess Senators note just House Reps (in which case statewide totals are important). I am just not sure how to balance all this...

Reply:
created #39